### ISSUE TITLE
Signage templates

### ISSUE THREAD

**Issue by syberphunk** [UKHackspaceFoundation/resources#5]

**Comment by syberphunk** [comment]
Talk on telegram suggested using the SVG examples on Hacman, else creating a program to create them

**Comment by syberphunk** [comment]
Relevant: http://www.online-sign.com/

**Comment by badspyro** [comment]
Yep, and the basics are on the hacman repo, and I've been playing with them recently - but I've had a fair number of comments about them (many non-specific), about everything from wording to having colours behind the text, to 'why do we need them and the suggestion not to wear gloves while on the pillar drill - that's a bit nanny state' (not quite, but not that big of an exaggeration). It's hard to tease out exactly what improvements are needed/useful.

Alas, I'm useless at CSS and any code in general. 

To some extent, it might be worth pulling in some more experienced 'Elf and Safety people who can give guidance.

I may have taken inspiration from that site you just posted...

**Comment by badspyro** [comment]
Here's the HacMan repo for the signage 

https://github.com/HACManchester/Health-And-Safety-Signage

**Comment by Axford** [comment]
Swindon ended up doing it by styled html:  https://rawgit.com/swindonmakers/documentation/master/signage/WarningSigns.html

**Comment by amunizp** [comment]
At my previous employment  we changed warnings with yellow background to white background as per new standard. http://www.hse.gov.uk/chemical-classification/labelling-packaging/hazard-symbols-hazard-pictograms.htm in case it helps.
